# DP: Disable -fauto-inc-dec by default for m68k
# DP: since it can generate ICEs in C++ code,
# DP: until a fix is found.

Index: b/src/gcc/common.opt
===================================================================
--- a/src/gcc/common.opt
+++ b/src/gcc/common.opt
@@ -863,7 +863,7 @@
 Generate unwind tables that are exact at each instruction boundary
 
 fauto-inc-dec
-Common Report Var(flag_auto_inc_dec) Init(1)
+Common Report Var(flag_auto_inc_dec) Init(0)
 Generate auto-inc/dec instructions
 
 ; -fcheck-bounds causes gcc to generate array bounds checks.
